sky/shell/shell.cc

```diff
@@ -45,6 +47,10 @@ Shell::~Shell() {
 }
 
 void Shell::Init(scoped_ptr<ServiceProviderContext> service_provider_context) {
+  CHECK(base::i18n::InitializeICU());
+#if !defined(OS_LINUX)
+  CHECK(gfx::GLSurface::InitializeOneOff());
```

Comment:
This is not even necessary on iOS and Mac. I was assuming Linux and Android needed the same. I suppose its only just EGL based surfaces